src/arch/riscv/rvk.hh

```diff
@@ -121,53 +121,89 @@ const uint8_t _rvk_emu_sm4_sbox[256] = {
     0xD7, 0xCB, 0x39, 0x48
 };
 
-inline int32_t _rvk_emu_sll_32(int32_t rs1, int32_t rs2)
-    { return rs1 << (rs2 & 31); }
-inline int32_t _rvk_emu_srl_32(int32_t rs1, int32_t rs2)
-    { return (uint32_t)rs1 >> (rs2 & 31); }
-inline int64_t _rvk_emu_sll_64(int64_t rs1, int64_t rs2)
-    { return rs1 << (rs2 & 63); }
-inline int64_t _rvk_emu_srl_64(int64_t rs1, int64_t rs2)
-    { return (uint64_t)rs1 >> (rs2 & 63); }
+inline int32_t
```

Comment:
The file is reference from [rvkrypto-fips](https://github.com/rvkrypto/rvkrypto-fips/tree/main). It should not apply clang-format